chain/default_syncer_test.go

```diff
@@ -317,11 +323,16 @@ func assertNoAdd(t *testing.T, chainStore chain.Store, cids types.SortedCidSet)
 	}
 }
 
-func requireHead(t *testing.T, chain chain.Store, head types.TipSet) {
+type headGetter interface {
```

Comment:
My intuition is that this should be used in the store test instead of redefining the same interface with the `requireHeadTipsetChainStore` name.

Reply:
It seems like redefining the same interface twice within the same package is too much, but please educated me otherwise. go-style interfaces are relatively new to me.

Reply:
No, you're right. I've combined the two into a single `HeadAndTipsetGetter` interface